app/src/main/java/com/example/aaa/pinkcalculator/Numbers.java

```diff
@@ -1,65 +1,87 @@
 package com.example.aaa.pinkcalculator;
 
+import android.widget.TextView;
+import static com.example.aaa.pinkcalculator.CommonConstants.MINUS;
+import static com.example.aaa.pinkcalculator.CommonConstants.POINT;
+import static com.example.aaa.pinkcalculator.CommonConstants.ZERO;
+
 public class Numbers {
+    private static final String MINUS_AND_NULL = "-0";
```

Comment:
actually name of this constant looks very strange and indicate that something wrong with your code.
also as Yahor mentioned
null is null
zero is 0